### PR TITLE
Epic Automation for ODC-4315

### PR DESCRIPTION
**Pull request by sanketpathak**
**Fix**: https://issues.redhat.com/browse/ODC-6387

**Test Setup**:

Cluster should be running on local

Check the TAGS in frontend/packages/pipilines/integration-tests/cypress.json file be: ""(@pre-condition or @smoke or @regression) not (@manual or @to-do or @un-verified or @broken-test)"",

Command to execute:

Run a cluster locally and execute the command `npm run test-cypress-pipelines` simultaneously

Execute the create-from-builder-page.feature file

**Browser**
Chrome 90

**Test Execution Screenshot:**
Screenshot from 2022-03-19 01-28-41

### CODE REVIEW

**Comment by rhamilto** [comment]
@sanketpathak, rebasing on master to pick up the changes in https://github.com/openshift/console/pull/11235 should help this get through the tests.

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature [hunk 1]
```diff
@@ -107,73 +107,57 @@ Feature: Create the pipeline from builder page
...
+                  | ptask-1       | kn        |
 
 
-        @regression @to-do
+        @regression
         Scenario: Upgrade tasks that are already installed on the cluster in pipeline builder page: P-02-TC09
             Given user is at Pipeline Builder page
              When user enters pipeline name as "pipeline-client"
-              And user clicks on Add task
+              And user installs and removes "openshift-client"
```
Comment (jrichter1) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature:142]: this should probably emphasize it is using the community version of the task, the red hat one does not offer multiple versions

Comment (jrichter1) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature:142]: this should probably emphasize it is using the community version of the task, the red hat one does not offer multiple versions

Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

**Comment by sanketpathak** [comment]
The changes only affect E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**